columnphysics/icepack_shortwave.F90

```diff
@@ -2654,8 +2652,11 @@ subroutine compute_dEdd (nilyr,    nslyr,    klev,  klevp,  &
                endif  !modal_aero
 !mgf--
                enddo       ! na
-               gaer = gaer/(waer+puny)
-               waer = waer/(taer+puny)
+               g(k)   = (g(k)*w0(k)*tau(k) + gaer) / &
+                             (w0(k)*tau(k) + waer)
+               w0(k)  =      (w0(k)*tau(k) + waer) / &
+                                   (tau(k) + taer)
+               tau(k) = tau(k) + taer
```

Comment:
It does look like these three lines will impact tr_aero and model_aero = .false. I looked back in the CICE5 code and this was the original implementation. Trying it out now.

Reply:
The changes to the "puny" implementation elsewhere will change answers but only slightly.  On the other hand, this block of changes was a bug and could be more consequential.